src/pages/Lend/Filter/SelectedRefinements.vue

```diff
@@ -184,26 +203,53 @@ export default {
 @import 'settings';
 
 .selected-refinements {
-	$fixed-row-height: 40;
+	$fixed-row-height: 38.4;
 
 	margin: 1rem 0;
 
-	.link {
-		color: $blue;
-		user-select: none;
+	.align-middle {
+		align-items: center;
+		display: -webkit-box;
```

Comment:
This should get automatically added in the scss compilation. (Do you have that turned off locally?)

Reply:
Same with align properties...

Reply:
@mcstover Thanks. Good catch!